cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -257,7 +257,7 @@ class ProfileCreateView extends Component {
       <Provider store={CreateProfileStore}>
         <div className="profile-create-view">
           <EntityTopPanel
-            title="Create a Google Dataproc Profile"
+            title="Create a profile for Google Cloud Dataproc"
```

Comment:
How come 'Google Cloud Dataproc' is hardcoded in the UI? Is there something missing from the backend?

Reply:
Good point. Now that we get the provisioner information from backend. Have updated.